src/services/authService.ts

```diff
@@ -0,0 +1,12 @@
+class AuthService {
+    public static authenticate(cb: any) {
```

Comment:
maybe rename cb as callback, would be slightly clearer, and we know the type of it, at least that it is a function.